File: XIScript/Class/BOInstanceGridPlaceholder.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace XIScript
{
    public class BOInstanceGridPlaceholder
    {

        private Dictionary<string, CBOPlaceholder> oMyBOInstances = new Dictionary<string, CBOPlaceholder>();

        public Dictionary<string, CBOPlaceholder> oBOInstances
        {
            get
            {
                return oMyBOInstances;
            }
            set
            {
                oMyBOInstances = value;
            }
        }

        private string sMyName;

        public string sName
        {
            get
            {
                return sMyName;
            }
            set
            {
                sMyName = value;
            }
        }
    }
}

File: XIAlgorithm/Class/BOInstanceGridPlaceholder.vb
Public Class BOInstanceGridPlaceholder

    Private oMyBOInstances As New Dictionary(Of String, CBOPlaceholder)
    Public Property oBOInstances() As Dictionary(Of String, CBOPlaceholder)
        Get
            Return oMyBOInstances
        End Get
        Set(ByVal value As Dictionary(Of String, CBOPlaceholder))
            oMyBOInstances = value
        End Set
    End Property

    Private sMyName As String
    Public Property sName() As String
        Get
            Return sMyName
        End Get
        Set(ByVal value As String)
            sMyName = value
        End Set
    End Property
End Class
